api/src/api.ts

```diff
@@ -112,6 +115,32 @@ export const plugin = {
       },
     });
 
+    server.route({
+      method: "GET",
+      path: "paper/{s2Id}",
+      handler: async (request) => {
+        const s2Id = request.params.s2Id;
+        const paper = await s2Api.getPaperUncached(s2Id, config.s2.apiKey)
+        .then(p => p.data)
+        .catch((error: AxiosError<S2ApiError>) => {
```

Comment:
Why use `.then()` and `.catch()` with `await`? May as well use `try/catch` instead, since it's easier to read.

Reply:
Old habit 👴 

try/catch would work better, yeah.